Dockerfile

```
@@ -0,0 +1,2 @@
+FROM squidfunk/mkdocs-material
+RUN pip install mkdocs-glightbox
```

```suggestion
RUN pip install -r requirements.txt
```

To be able to add more requirements/packages in the future AND not having to keep this file up-to-date.

---

Also, you could add a "Using Docker"-section to the main README.
Because when using this Docker-image/container, it isn't necessary to use the Python-virtualenv.

(But both approaches remain useful, for when someone isn't able/allowed to install Docker)